server/controllers/alertsController.ts

```diff
@@ -272,10 +268,13 @@ export default class AlertsController {
 
     // Initialise form
     const alertFlash = req.flash('alert')
-    const { description } = alertFlash?.length ? (alertFlash[0] as never) : { description: alert.description }
+    const { description, activeTo } = alertFlash?.length
+      ? (alertFlash[0] as never)
```

Comment:
Why would that be as never?

Reply:
Typescript allows you to use `never` similar to `any` in some situations where you just don't care what the type is, but agree it's more useful to actually describe the type properly so have updated